shared/fs/filepreview/default-view.js

```diff
@@ -72,20 +72,6 @@ const DefaultView = (props: DefaultViewProps) => (
         onClick={props.onDownload}
       />
     )}
-    {// We only show this button for files with no extensions, because our
-    // mime type list cannot be exaustive. For example it'd be weird to show
-    // an Illustrator file as plain text.
```

Comment:
So is this no longer true? Are we opening them as text based on the results of `DetectContentType` now?

Reply:
Yeah I think so; files with no extension would usually get a mime type from that, and we'd have chosen a different viewer (`TextView` being one of the possibles) if we can show them. If we reach here, it must have been a mime type that's not `text/plain`, which we shouldn't throw into ` webview anyway.